tests/trajectory.py

```diff
@@ -99,14 +90,9 @@ def test_close(self):
 
     def test_write(self):
         frame = Frame()
-        frame.resize(4)
```

Comment:
Why is resize now a bad idea?

Reply:
It's not that it is a bad idea, more that we can now directly add atoms to the frame, without creating a separated topology first.